classes/class-gf-securesubmit.php

```diff
@@ -885,10 +885,10 @@ public function get_submission_dataACH($feed, $form, $entry) {
 
         $accountType = rgpost(GF_Field_HPSach::HPS_ACH_TYPE_FIELD_NAME);
         $checkType = rgpost(GF_Field_HPSach::HPS_ACH_CHECK_FIELD_NAME);
-        $accountTypeOptions = array(
+        $accountTypeOptions = array( 1=>
             HpsAccountType::CHECKING,
             HpsAccountType::SAVINGS);
-        $checkTypeOptions = array(
+        $checkTypeOptions = array(1=>
```

Comment:
Why start the index here and above at `1`? Why not just change the options available on the form? If you want to keep it the way it is, I would advise to set the key on the other values in the array as well.